src/views/LayoutWithRouting.tsx

```diff
@@ -28,10 +28,18 @@ const SIDENAVBAR_ITEMS: NavItemType[] = [
   },
 ]
 
+export type RouteComponentProps = {
```

Comment:
Do we even need that now? This was previously enforced by `reach`, I think we can just remove that